README.md

```diff
@@ -17,11 +17,11 @@ Alacritty currently supports macOS, Linux, BSD, and Windows.
 ## About
 
 Alacritty is a terminal emulator with a strong focus on simplicity and
-performance. Bloated features are avoided, and the features we do include are
-carefully considered. This means you won't find things like tabs or splits
-(which are best left to a window manager or [terminal multiplexer][tmux]) nor
-niceties like a GUI config editor. We also strive to keep alacritty faster than
-any other terminal available. Performance matters!
+performance. Bloated features are avoided, and the included features are
+carefully considered. With such a strong focus on performance, you can expect
+Alacritty to be blazing fast, always. Alacritty should also "just work" by
+making sane choices in defaults, but allowing [configuration](#configuration)
+of many aspects of the terminal.
```

Comment:
This might just be me, but the words `Bloated features are avoided` sound a bit judgemental torwards other terminals and in general the word `Bloated` feels very opinionated.

I'd propose something like this:
```
With such a strong focus on performance, included features are carefully considered and you can expect Alacritty to always be blazingly fast.
```

I think the content of the last sentence is great and a welcome addition, however I'm not sure if I'd put something like `"just work"` in a manpage (in the readme it would be fine I think, but reusability would be nice).

Maybe something like this?
```
By choosing sane defaults, Alacritty should also be pleasant to use out of the box, while still allowing [configuration](#configuration) of many aspects of the terminal.
```

I'm not sure if this is an improvement, so let me know what you think please.

Reply:
Slight nit on the flow of your first proposition (moved `always` before `expect`):
```
With such a strong focus on performance, included features are carefully considered and you can always expect Alacritty to be blazingly fast.
```

I like your second proposition regarding the configuration wording,  though.

Reply:
I like that!

Reply:
Yep, that basically fixes the problem I had reading it. I'll update these all in a bit.

Reply:
Any feedback on my proposal to change the "just works" section @nixpulvis? I think it's a bit colloquial.